### ISSUE TITLE
Loot distibution 

### ISSUE THREAD

**Issue by SmurferSurfer22** [Roeshambo/MonolithDKP#377]
Is there a way to make loot distribution and the biding work after you pull the loot off the boss?

**Comment by BreakBB** [comment]
@SmurferSurfer22 Yes absolutetly. You can use the `/dkp bid <ItemLink>` (use Shift + Click on an item for the ItemLink) command for this. We are always collecting the boss loot and hand it out while continueing with the raid.
